vnext/docs/api/react-native-windows.iviewwindowsprops.md

```diff
@@ -10,6 +10,12 @@
 export interface IViewWindowsProps extends IKeyboardProps, ViewProps 
 ```
 
+## Remarks
```

Comment:
I don't know which tool changed it automatically.

Reply:
yarn build will run api-documenter which will gen this.  Not sure why it changed with your PR.